components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.cpp

```diff
@@ -117,10 +117,21 @@ class RFPins {
     RFPins(PinName spi_sdi, PinName spi_sdo,
            PinName spi_sclk, PinName spi_cs, PinName spi_sdn,
            PinName spi_gpio0, PinName spi_gpio1, PinName spi_gpio2,
-           PinName spi_gpio3);
+           PinName spi_gpio3
+#if MBED_CONF_S2LP_EXTERNAL_FRONTEND
```

Comment:
Driver functionality should not be based on config options. It should always be possible to specify all pins and functionality manually from application C++ code, as it is now.

The config options should only be controlling the "default" behaviour. The config options should work like the other ones and only be used in `NanostackRfPhy::get_default_instance`.

Reply:
ok